.github/workflows/build-docker-typesafe.yml

```diff
@@ -0,0 +1,55 @@
+name: Fineract Docker build - Type-Safe project
+
+on:
+  push:
+    branches:
+      - FINERACT-2021/refactor_GET_Methods
```

Comment:
In your main Jira ticket you can use sub-tasks to address any partial goals/work... that way you can always use a proper Jira ticket name ("FINERACT-1234") to name your branches and don't have to invent names... we have all the necessary details anyway on the ticket(s). Just saying for the next time, makes your life easier.

Reply:
Noted